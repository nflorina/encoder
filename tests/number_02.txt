-111
-1379
119151
1133
-1966421
1666632
-16
1231388
111
1248
-168
-113951
-11
-125472
1338799
-1726
-147979
-164557
-129
191
13419115
13298
18645
-14379
1413561
12817283
1714
16144
-1929671
-131
-13
-114297
1164
-117
1665837
135
182
-18
-18
-1617863
-112115
-111
-196165
199
1196274
19128792
165987
183
-1979417
14316538
-1111
12856414
-1846
-1511919
-18
-12962
191
-121
12
1229
14831723
19334369
-178187
-14258275
-192
-11
129
-1223
-19891
1
END
