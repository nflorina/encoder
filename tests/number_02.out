0 0 69
11113799151113311119664266663211688123131118124168111395111254729913387172614797916455712991191151341981328645114379611413583128172714161441119296711313114297641111783716655138211818161786311211511116519699196274119219128798716583117197948143165311118564141218461191915181296291111221912283172314933436911781871425827519211912122311989
11113799151113311119664266663211688123131118124168111395111254729913387172614797916455712991191151341981328645114379611413583128172714161441119296711313114297641111783716655138211818161786311211511116519699196274119219128798716583117197948143165311118564141218461191915181296291111221912283172314933436911781871425827519211912122311989
